script/smt-mirror-sle9

```diff
@@ -6,6 +6,9 @@
 
 use strict;
 use warnings;
+use FindBin;
+BEGIN { unshift @INC, "$FindBin::Bin/../www/perl-lib" }
+
 use Config::IniFiles;
```

Comment:
@jsrain what do you think? It is time now to drop also SLES9 mirroring? I would say this would be a good time to do it.